### DISCUSSION TITLE
Getlookup function

### DISCUSSION THREAD

**Discussion by havvanur160401046** [serenity-is/Serenity#5806] [Help]
![image](https://user-images.githubusercontent.com/24252615/122043363-ef3db100-cde3-11eb-87dc-32be2743b68c.png)

I add lookup function but it is deleted after running the project

**Comment by eissa1005** [comment]
@havvanur160401046
Don't add it yourself but use attribute [LookupScript] in Row will add it
If you want to access any column, use LookupInclude

![Untitled](https://user-images.githubusercontent.com/20481378/122055323-cc61cb80-cde8-11eb-96a6-519e9367ff7b.png)
